smt/applications/ego.py

```diff
@@ -403,7 +428,7 @@ def _find_best_point(self, x_data=None, y_data=None, enable_tunneling=False):
             if not success:
                 self.log("New start point for the internal optimization")
                 n_optim += 1
-
+                n_start += 10
```

Comment:
Why do you increase the `n_start` ? I mean, as `n_start` and even `n_max_optim` are controlled by the user to achieve give optimization more chances, is it required?